.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,16 @@
+# pre-commit hook definition
+# See http://pre-commit.com/ for more info
+#
+# To use, add the following to .pre-commit-config.yaml in your repo:
+# -   repo: git@github.com:vvakame/typescript-formatter.git
+#     sha: e3200b7873d03e7b13b7f8d0bd5cb760ed400585
+#     hooks:
+#     -   id: tsfmt
+
+-   id: tsfmt
+    name: TypeScript Formatter (tsfmt)
+    description: TypeScript code auto-formatter.
+    entry: tsfmt
+		args: ['--verify']
+    language: system
```

Comment:
Why do you use `system`?
Is `node` more appropriate? http://pre-commit.com/#node

My suggestions, We should use project local tsfmt, not system wide tsfmt.
I think you have a two or more project about tsfmt using. There project should be able to update dependencies independently.